superset/config.py

```diff
@@ -873,7 +873,8 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # query costs before they run. These EXPLAIN queries should have a small
 # timeout.
 SQLLAB_QUERY_COST_ESTIMATE_TIMEOUT = int(timedelta(seconds=10).total_seconds())
-# The feature is off by default, and currently only supported in Presto and Postgres.
+# The feature is off by default, and currently only supported in Presto and Postgres,
+# and Bigquery.
```

Comment:
The current config, docs and the examples in `config.py` are in a pretty horrible state right now. I observed the following:

1. The `ESTIMATE_QUERY_COST` config is in fact a feature flag and needs to be moved to `DEFAULT_FEATURE_FLAGS`: https://github.com/apache/superset/blob/49f1cfc3f99478a3394fccddc50ab9bf7f5c4ae4/superset/config.py#L879 This should be updated
2. This example is broken: https://github.com/apache/superset/blob/49f1cfc3f99478a3394fccddc50ab9bf7f5c4ae4/superset/config.py#L905 . It should in fact be `# QUERY_COST_FORMATTERS_BY_ENGINE = {"postgresql": postgres_query_cost_formatter}`. This should also be updated.
3. There are no docs for this feature. In the deprecated docs there's a SQL Lab section that seems to have been lost over the years, and the content is also incorrect: https://apache-superset.readthedocs.io/en/latest/sqllab.html I don't expect you to add docs for this, but just mentioning this here to call attention to it. FYI @rusackas

Reply:
Thanks for the callout... putting it on my very long to-do list!

Reply:
Fixed the things and added some more comments.

About the docs I would love to fix the content. Only thing is that the superset/blob/master/docs/sqllab.rst file doesnt even exist now.

Reply:
@zamar-roura let's leave the docs for a follow-up PR 👍